run_e2e.sh

```diff
@@ -42,13 +42,16 @@ ginkgo_args=(
     "--noColor=true"
 )
 
+# Number of nodes in the cluster.
+NUM_NODES="${NUM_NODES:-$(kubectl describe nodes | grep '^Taints:' | grep -vF 'node-role.kubernetes.io/master:NoSchedule' | wc -l)}"
```

Comment:
Hi @neolit123,

Thank you for catching the previous error. Would you please take a look at this? Thank you!

Reply:
see https://github.com/kubernetes/kubernetes/pull/74854/files#diff-0efd26687540ba2cbfd9547019622551R63

Reply:
Per Slack, Lubomir noted:

> the list under `Taints:` can span on multiple lines.

Reply:
Thank you @neolit123.